data/hai/hai missions.txt

```diff
@@ -324,6 +324,11 @@ mission "Returning Home"
 				to display
 					has "hai space start"
 			`	"Good. My name is Elliot." You introduce yourself to Elliot and shake hands.`
+				to display
+					has "human space start"
+			`	He sighs. "I was hoping to get someone from human space. Well, a captain's a captain, I guess. My name is Elliot." You introduce yourself to Elliot and shake hands.
+				to display
+					has "hai space start"
```

Comment:
Shouldn't this be `not "human space start"` or something? In case a plugin has some other start neither option would display.

Reply:
```suggestion
					not "human space start"
```
I agree, but this is probably based on him recognizing your hai style clothing? idk

Reply:
Good catch, thanks.